src/System.ServiceModel.Duplex/pkg/System.ServiceModel.Duplex.pkgproj

```diff
@@ -6,7 +6,7 @@
       <SupportedFramework>net45;win8;netcoreapp1.0;$(AllXamarinFrameworks)</SupportedFramework>
```

Comment:
You retargeted the reference assembly but didn't update the supportedFramework entries.